torch/quantization/qconfig.py

```diff
@@ -66,6 +66,8 @@ def __new__(cls, activation=torch.nn.Identity, weight=torch.nn.Identity):
                                          weight=default_weight_observer)
 float16_dynamic_qconfig = QConfigDynamic(activation=PlaceholderObserver.with_args(dtype=torch.float32),
                                          weight=PlaceholderObserver.with_args(dtype=torch.float16))
+float16_static_qconfig = QConfigDynamic(activation=PlaceholderObserver.with_args(dtype=torch.float16),
```

Comment:
are we treating fp16 emulation as static quantization?  Thoughts about having it be a new quantization type instead?  cc @raghuramank100

Reply:
yes, I think this falls into the category of static quant, (static, dynamic, weight_only are referring to how we quantize activations)